wgsl/index.bs

```diff
@@ -4264,12 +4353,14 @@ The interface includes:
 * Pipeline inputs and outputs
 * Buffer resources
 * Texture resources
+* Sampler resources
 
 These objects are represented by module-scope variables in certain [=storage classes=].
 
 We say a variable is <dfn noexport>statically accessed</dfn> by a function if any subexpression
 in the body of the function uses the variable's identifier,
 and that subexpression is [=in scope=] of the variable's declaration.
+Static access of a `let`-declared constant is defined similarly.
```

Comment:
Do we want to define static access for *all* things, like function-local `let` expressions?
It seems to me that for all the purposes where the "static access" is needed, we could define it only on module-scope items:
  - functions reachable from the entry point
  - resources, constants, variables

It seems to me that there is no need to try defining what "static access" means within any function. Instead, we consider all of the things accessed if a function is in the call graph.

Reply:
Right.  But it takes more words to carve out just the necessary cases.
And having to express an extra condition means the reader has to think just a bit more to get a correct idea of the situation.
I prefer to keep this as it is.

To be clear about it, I needed to add a clause for let-declared constants so that I could describe the totality of the footprint of what comprises a shader.